### PR TITLE
Fix/wrap in actions

### PR DESCRIPTION
**Pull request by enidemi**
**Problem:**
There are some problems with the wrap-in actions. First is there are 2 of them, mostly the same code but one of them is not updated that much for example only one handles fragments. Inside the wrap-in update where the insertion happens it is hard to follow, the nested if-elses makes it quite error prone. The result is often not good, for example wrapping in elements with flexGrow just makes them zero size, or wrapping even just absolute element causes shifts.

**Fix:**
Remove `wrapInView`, only keep `wrapInElement`. Updated `wrapInElement` to use insertion commands for simple jsx elements, and replaced the if-elses with switch cases, it's easier to see if all insertion target cases are handled, next time we add a new type of InsertionPath this will break.

Next step is the wrapper element frame/position etc. based on where the insertion happens. The current PR keeps the default divs and elements as wrappers, so it's still not good.

**Commit Details:**
- added basic tests for wrap-in simple elements
- found tests for `wrapInView` context-menu item too, I grouped them into one `describe`
- renamed `unwrap-conditional-helpers` to `wrap-unwrap-helpers`, and moved the wrapping insertion here.
- removed `getIsFlexBasedOnName_KILLME_EXPERIMENTAL` and its friends from the floating-menu

### CODE REVIEW

**Review by bkrmendy — APPROVED** [review]

File: editor/src/components/canvas/ui/floating-insert-menu.tsx [hunk 1]
```diff
@@ -160,26 +159,6 @@ function useGetInsertableComponents(): InsertableComponentFlatList {
...
 }
 
-function getIsFlexBasedOnName_KILLME_EXPERIMENTAL(name: JSXElementName): boolean {
-  return (
-    name.propertyPath.propertyElements.length === 0 &&
-    (name.baseVariable === 'FlexRow' || name.baseVariable === 'FlexCol')
-  )
-}
-
-function getIsFlexDirectionBasedOnName_KILLME_SERIOUSLY_EXPERIMENTAL(
```
Comment (Rheeseyb) [inline editor/src/components/canvas/ui/floating-insert-menu.tsx:170]: https://github.com/concrete-utopia/utopia/issues/1580!

File: editor/src/components/editor/actions/actions.spec.browser2.tsx [hunk 2]
```diff
@@ -1121,4 +1127,111 @@ describe('actions', () => {
       )
     })
   })
+  describe('WRAP_IN_ELEMENT', () => {
+    it(`Wraps in 2 elements`, async () => {
```
Comment (Rheeseyb) [inline editor/src/components/editor/actions/actions.spec.browser2.tsx:1131]:
```suggestion
    it(`Wraps 2 elements`, async () => {
```

File: editor/src/components/editor/actions/actions.spec.browser2.tsx [hunk 3]
```diff
@@ -1121,4 +1127,111 @@ describe('actions', () => {
...
+          `<div data-uid='aaa' style={{contain: 'layout', width: 300, height: 300}}>
+            <div style={{backgroundColor: '#aaaaaa33', position: 'absolute'}} data-uid='${testUID}'>
+              <div data-uid='ccc' style={{position: 'absolute', left: 20, top: 50, bottom: 150, width: 100}} />
+              <div data-uid='ddd' style={{width: 60, height: 60}} />
+            </div>
+          </div>`,
+        ),
+      )
+    })
+    it(`Wraps in 2 elements inside a flex layout`, async () => {
```
Comment (Rheeseyb) [inline editor/src/components/editor/actions/actions.spec.browser2.tsx:1164]:
```suggestion
    it(`Wraps 2 elements inside a flex layout`, async () => {
```

File: editor/src/components/editor/actions/actions.spec.browser2.tsx [hunk 4]
```diff
@@ -1121,4 +1127,111 @@ describe('actions', () => {
...
+              <div style={{backgroundColor: '#aaaaaa33', position: 'absolute'}} data-uid='${testUID}'>
+                <div data-uid='ddd' style={{flexGrow: 1, height: '100%'}} />
+                <div data-uid='eee' style={{width: 100, height: 60}} />
+              </div>
+            </div>
+          </div>`,
+        ),
+      )
+    })
+    it(`Wrap in 2 elements with a fragment`, async () => {
```
Comment (Rheeseyb) [inline editor/src/components/editor/actions/actions.spec.browser2.tsx:1203]:
```suggestion
    it(`Wrap 2 elements with a fragment`, async () => {
```

File: editor/src/components/editor/actions/wrap-unwrap-helpers.tsx [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+            editor,
+            staticTarget,
+            elementToInsert,
+            importsToAdd,
+          )
+          return { updatedEditor: withTargetAdded, newPath: newPath }
+        default:
+          const _exhaustiveCheck: never = staticTarget
+          return { updatedEditor: editor, newPath: null }
+      }
```
Comment (Rheeseyb) [inline editor/src/components/editor/actions/wrap-unwrap-helpers.tsx:289]: It looks like this block of code is identical for the 3 cases?

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/components/canvas/ui/floating-insert-menu.tsx (hunk 1 above)
Comment (enidemi) [reply editor/src/components/canvas/ui/floating-insert-menu.tsx:170]: yes, it's time to fix that :)
